pandas/tests/frame/test_apply.py

```diff
@@ -880,11 +881,14 @@ def f():
             with np.errstate(all='ignore'):
                 df.agg({'A': ['abs', 'sum'], 'B': ['mean', 'max']})
 
-    def test_transform_abs_name(self):
+    @pytest.mark.parametrize('method', [
+        'abs', 'shift', 'pct_change', 'cumsum', 'rank',
+    ])
+    def test_transform_method_name(self, method):
         # https://github.com/pandas-dev/pandas/issues/19760
         df = pd.DataFrame({"A": [-1, 2]})
-        result = df.transform('abs')
-        expected = pd.DataFrame({"A": [1, 2]})
+        result = df.transform(method)
+        expected = operator.methodcaller(method)(df)
```

Comment:
Learned something new today. Just out of curiosity, is this any different from doing `getattr(df, method)()`?

Reply:
in this case, no. methodcaller is nicer though when you need to bind parameters, but don't have the object yet (common in pytest fixtures).